src/PrestaShopBundle/Translation/PrestaShopTranslatorTrait.php

```diff
@@ -130,6 +130,16 @@ public function transChoice($id, $number, array $parameters = array(), $domain =
         return vsprintf(parent::transChoice($id, $number, array(), $domain, $locale), $parameters);
     }
 
+    /**
+     * @param $locale
+     *
+     * @return bool
+     */
+    public function isCatalogLoaded($locale)
+    {
+        return isset($this->catalogues[$locale]);
```

Comment:
Shouldn't it be better to check for emptyness?
```suggestion
        return empty($this->catalogues[$locale]);
```